### PR TITLE
Cleaning up a bunch of migrations, also adding property photos model

### PR DESCRIPTION
**Pull request by yilunc**
Removing a bunch of unnecessary models and fixing some broken associations. Tested locally to be fully working now. Also adding a new model for property photos.

@Investify/everybody

### CODE REVIEW

File: db/migrate/20160820185418_add_admin_to_users.rb [hunk 1]
```diff
@@ -1,5 +0,0 @@
-class AddAdminToUsers < ActiveRecord::Migration
-  def change
-    add_column :users, :admin, :boolean
-  end
-end
```
Comment (seancannedham) [inline db/migrate/20160820185418_add_admin_to_users.rb]: we'll need this one

File: db/migrate/20160820190245_delete_boolean_from_users.rb [hunk 2]
```diff
@@ -1,4 +0,0 @@
-class DeleteBooleanFromUsers < ActiveRecord::Migration
-  def change
-  end
-end
```
Comment (seancannedham) [inline db/migrate/20160820190245_delete_boolean_from_users.rb]: @qasa1 what will this be used for?

File: db/migrate/20160820190429_delete_admin_from_users.rb [hunk 3]
```diff
@@ -1,4 +0,0 @@
-class DeleteAdminFromUsers < ActiveRecord::Migration
-  def change
-  end
-end
```
Comment (seancannedham) [inline db/migrate/20160820190429_delete_admin_from_users.rb]: @qasa1 and this?

File: db/migrate/20160820202415_add_userid_to_users.rb [hunk 4]
```diff
@@ -1,5 +0,0 @@
-class AddUseridToUsers < ActiveRecord::Migration
-  def change
-    add_column :users, :user_id, :integer
-  end
-end
```
Comment (seancannedham) [inline db/migrate/20160820202415_add_userid_to_users.rb]: @qasa1 do we need this? and the delete?

File: db/migrate/20160820225351_add_approved_to_users.rb [hunk 5]
```diff
@@ -1,5 +0,0 @@
-class AddApprovedToUsers < ActiveRecord::Migration
-  def change
-    add_column :users, :approved, :boolean
-  end
-end
```
Comment (seancannedham) [inline db/migrate/20160820225351_add_approved_to_users.rb]: @qasa1 was this supposed to be adding approval to properties?

File: db/migrate/20160820185418_add_admin_to_users.rb (hunk 1 above)
Comment (yilunc) [inline db/migrate/20160820185418_add_admin_to_users.rb]: fixed by adding the field in the `users` table migration!

**Comment by seancannedham** [comment]
@qasa1 I tagged you in a few files, are any of these or the other ones needed? If so we won't delete

File: db/migrate/20160820190245_delete_boolean_from_users.rb (hunk 2 above)
Comment (qasa1) [inline db/migrate/20160820190245_delete_boolean_from_users.rb]: delete it

File: db/migrate/20160820190429_delete_admin_from_users.rb (hunk 3 above)
Comment (qasa1) [inline db/migrate/20160820190429_delete_admin_from_users.rb]: delete it

File: db/migrate/20160820202415_add_userid_to_users.rb (hunk 4 above)
Comment (qasa1) [inline db/migrate/20160820202415_add_userid_to_users.rb]: lol this was a mistake. "user_id to users table". what an idiot I am. I meant to add it to properties table.

Comment (qasa1) [inline db/migrate/20160820202415_add_userid_to_users.rb]: so delete it

File: db/migrate/20160820225351_add_approved_to_users.rb (hunk 5 above)
Comment (qasa1) [inline db/migrate/20160820225351_add_approved_to_users.rb]: yes this is for approval form. If you want it. keep it otherwise delete

**Pull request merged**